lib/model/store.dart

```diff
@@ -323,6 +323,25 @@ abstract class GlobalStore extends ChangeNotifier {
       zulipFeatureLevel: Value(data.zulipFeatureLevel)));
   }
 
+  Future<void> updateRealmData(int accountId, {
+    required String realmName,
+    required Uri realmIcon,
+  }) async {
+    assert(_accounts.containsKey(accountId));
```

Comment:
There's an async gap before this caller, and normally we consider it possible that the account was logged out during an async gap. But I guess in this case, the async gap is for updating an account (`await globalStore.updateZulipVersionData`), and that can't successfully happen concurrently with deleting the account? Meaning this `assert` should actually always hold?